final/problem-def-ana/problem-ana.tex

```diff
@@ -92,30 +74,39 @@ \subsubsection{Language specific additional commands}
 language is a DSL. It is clear therefore that a different approach
```

Comment:
This subsentence (`, for example, when the language is a DSL`) should be removed.